bin/layerlens.dart

```diff
@@ -57,6 +59,11 @@ void main(List<String> args) async {
       defaultsTo: null,
       help: 'Package name, that is needed when internal '
           'libraries reference each other with `package:` import.',
+    )
+    ..addMultiOption(
+      _Options.buildFilter.name,
+      help:
```

Comment:
Can you add link to the section 'Build filters' in README.md'?

Reply:
added